site/docs/snapshot-tag-branch.md

```diff
@@ -0,0 +1,148 @@
+# Snapshot Tagging and Branching
+
+Iceberg snapshot tagging and branching feature offers user a Git-like experience in manging table snapshots.
```

Comment:
I agree that the `git-like experience` part could confuse users as to what that really means.

I believe in some version of the design doc you named it `Named Snapshots` (or something similar) as to me that's what this feature really is. Those are named snapshots where the `type` can be either a `branch` (moves forward with changes) or a `tag` (does not move forward) and I would probably rather explain this feature from this perspective

Reply:
Agree, I updated it, please let me know what you think about the new wording.